source/common/grpc/common.cc

```diff
@@ -105,12 +105,12 @@ bool Common::resolveServiceAndMethod(const Http::HeaderEntry* path, std::string*
   if (path == nullptr || path->value().c_str() == nullptr) {
     return false;
   }
-  std::vector<std::string> parts = StringUtil::split(path->value().c_str(), '/');
+  auto parts = StringUtil::splitToken(path->value().c_str(), "/");
   if (parts.size() != 2) {
     return false;
   }
-  *service = std::move(parts[0]);
-  *method = std::move(parts[1]);
+  *service = std::move(std::string(parts[0]));
```

Comment:
You could also
  service->assign(parts[0].data(), parts[0].size());
not sure which is better; either is fine with me.  Just thought I'd point out the alternative form.